src/CMakeLists.txt

```diff
@@ -10,13 +10,12 @@ if(nfd_PLATFORM STREQUAL PLATFORM_WIN32)
   list(APPEND SOURCE_FILES nfd_win.cpp)
 endif()
 
-if(nfd_PLATFORM STREQUAL PLATFORM_LINUX)
+if(nfd_PLATFORM STREQUAL PLATFORM_UNIX)
   find_package(PkgConfig REQUIRED)
   # for Linux, we support GTK3 and xdg-desktop-portal
   option(NFD_PORTAL "Use xdg-desktop-portal instead of GTK" OFF)
   if(NOT NFD_PORTAL)
-    pkg_check_modules(GTK3 REQUIRED gtk+-3.0)
-    message("Using GTK version: ${GTK3_VERSION}")
```

Comment:
Why remove this but keep the `message("Using DBUS version: ${DBUS_VERSION}")` message for the portal case below?  This message is meant to be here so that users can see that they are using the correct backend.

Also, should we make similar modifications for configuring portals too (i.e. make the same changes for the `dbus-1` package)?

Reply:
I'm not sure why I removed that message. I put it back.